cob_default_robot_config/cob4-1/command_gui_buttons.yaml

```diff
@@ -4,45 +4,13 @@
 #  buttons: [[button_name1,function_name1,parameter_name2,
 #    [button_name2,function_name2,parameter_name2]]
 
-group1:
+group10:
```

Comment:
in the command_gui the list is just sorted alphabetically, which means the following order

group1
group11
group1000000043
group2
group3

Switching to two-digit groups is OK for me